src/main/java/org/jabref/logic/autosaveandbackup/BackupManager.java

```diff
@@ -176,7 +177,9 @@ private void performBackup(Path backupPath) {
         SavePreferences savePreferences = preferences.getSavePreferences()
                                                      .withMakeBackup(false);
         Charset encoding = bibDatabaseContext.getMetaData().getEncoding().orElse(StandardCharsets.UTF_8);
-        try (Writer writer = Files.newBufferedWriter(backupPath, encoding)) {
+        // We want to have successful backups only
+        // Thus, we do not use a plain "FileWriter", but the "AtomicFileWriter"
+        try (Writer writer = new AtomicFileWriter(backupPath, encoding, false)) {
```

Comment:
But doesn't that will result in unusable backup files on crashes if the atomic moving of the file is stopped in between? I thought that's what we wanted to avoid...

Reply:
When the writing itself fails (e.g., an exception), we also have an unusable file. My bet is that it is more likely that writing fails instead that the moving fails.

Reply:
What happens if you hard power off your machine (or kill the jabref process) during the write of the backup? Doesn't this create a broken backup file that then jabref wants to "restore" from?

Reply:
Which is another pro for the "AtomicWriter". I will add it to the comment there.

Reply:
But the writer itself is still broken. uses the wrong outputstream

Reply:
Pleas keep in mind that the writer does not really prevent others from writing to the same file